run_tests.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 
-if __name__ == '__main__':
-    suite = unittest.defaultTestLoader.discover('task_manager/tests')
+if __name__ == "__main__":
+    suite = unittest.defaultTestLoader.discover("task_manager/tests")
```

Comment:
why double quotes over single quotes?

Reply:
my linter probably did this.

Reply:
please use the linter that we use to build